pkgs/tools/misc/coreutils/default.nix

```diff
@@ -16,7 +16,8 @@
 , minimal ? true
 , withOpenssl ? !minimal, openssl
 , withPrefix ? false
-, singleBinary ? "symlinks" # you can also pass "shebangs" or false
+, singleBinary ? if stdenv.isFreeBSD then false else "symlinks" # you can also pass "shebangs" or false
+, withAutoreconf ? true,
```

Comment:
We shouldn't add such an option.

Reply:
Don't we need to disable this on stdenv bootstrapping? I don't think our bootstrapping tarball has enough dependencies to have this.

Reply:
This is an incredibly old diff, from back when I was doing the impure bootstrap. it's distinctly possible that we can use autoreconf now. I will try.

Reply:
It is removed now.